packages/lib-classifier/src/plugins/drawingTools/models/marks/Point/Point.js

```diff
@@ -23,8 +23,8 @@ const PointModel = types
       const theta = (DELETE_BUTTON_ANGLE) * (Math.PI / 180)
       const dx = (20 / scale) * Math.cos(theta)
       const dy = -1 * (20 / scale) * Math.sin(theta)
-      const x = self.x + dx
-      const y = self.y + dy
+      const x = dx
+      const y = dy
```

Comment:
Should there be changes to Point in a PR for Ellipse?

Reply:
Yep.

Reply:
Looking back through the commit history, this is also because transforms are handled by the drawing tool root. The delete button position is now relative to `0, 0`.